addons/website_event_track/static/src/js/service_worker.js

```diff
@@ -61,8 +61,13 @@ const isCacheFull = async () => {
     if (!("storage" in navigator && "estimate" in navigator.storage)) {
         return false;
     }
-    const { usage, quota } = await navigator.storage.estimate();
-    return usage / quota > MAX_CACHE_QUOTA || usage > MAX_CACHE_SIZE;
+    try {
+        const { usage, quota } = await navigator.storage.estimate();
```

Comment:
maybe only protect this line with the try/catch?

Reply:
But actually the following line (`return ...`) doesn't make sense without this call, so... :shrug:

Reply:
indeed, but if for some reason the return line crashes, the error will be caught